src/cli/peakdetector/PeakDetectorCLI.cpp

```diff
@@ -353,8 +354,13 @@ void PeakDetectorCLI::processPeaksArgsXML(xml_node& peaksArgs) {
 		}
 		else if (strcmp(node.name(),"ppmMerge") == 0) {
 
-			mavenParameters->ppmMerge = atof(node.attribute("value").value());
-
+			mavenParameters->massCutoffMerge->setMassCutoffAndType(atof(node.attribute("value").value()),"ppm");
+			mavenParameters->compoundMassCutoffWindow->setMassCutoffType("ppm");
```

Comment:
As we know in case of testing, current xml (test.xml) file for parameters setting in (..bin/methods) folder does not have (compoundPPMWindow) parameter, it just has (ppmMerge). I have just set unit for (compoundMassCutoffWindow) as unit of (ppmMerge). This line has to be removed if mDa opetion is introduced in testing. I have given a comment in code base already.